use-cases/model-fine-tuning-pipeline/data-processing/ray/src/Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM python:3.12.8-slim-bullseye as build-stage
+FROM python:3.12.4-slim as build-stage
```

Comment:
Use `-slim-bookworm`

Reply:
Any reason we went with `3.12` instead of `3.13`?

Reply:
yes, also the python minor version on Ray cluster and the client must be the same. If not, we need to pass RAY_IGNORE_VERSION_MISMATCH as runtime env. Ray cluster is using 2.40.0-py312-gpu so stuck we 3.12 on the client side.